crawl-ref/source/mon-pick-data.h

```diff
@@ -393,21 +393,59 @@ static const vector<pop_entry> population[] =
 },
 
 { // Spider Nest
-  { -1,  4,  260, RISE, MONS_HORNET },
-  { -1,  6,   89, SEMI, MONS_BOULDER_BEETLE },
-  { -1,  9,  765, SEMI, MONS_DEMONIC_CRAWLER },
-  {  1,  4,  955, FLAT, MONS_WOLF_SPIDER },
-  {  1,  4,  985, FLAT, MONS_JUMPING_SPIDER },
-  {  1,  4,  910, FLAT, MONS_TARANTELLA },
-  {  1,  4, 1045, FLAT, MONS_REDBACK },
-  {  1,  4,  550, FLAT, MONS_ORB_SPIDER },
-  {  1,  6,  100, FLAT, MONS_TORPOR_SNAIL },
-  {  1,  6,  190, SEMI, MONS_EMPEROR_SCORPION },
-  {  1,  6,  155, SEMI, MONS_MOTH_OF_WRATH },
-  {  1,  9,  300, SEMI, MONS_ENTROPY_WEAVER },
-  {  1,  9,  200, PEAK, MONS_GHOST_MOTH },
-  {  2,  4,   12, FLAT, MONS_MELIAI },
-  {  2, 10,  260, PEAK, MONS_SPARK_WASP },
+  // Easy enemies:
+  {  1,  4,   200, FLAT, MONS_BOULDER_BEETLE },
+  {  1,  4,   100, FALL, MONS_BOULDER_BEETLE },
+  {  1,  4,   800, FLAT, MONS_REDBACK },
+  {  1,  4,   800, FALL, MONS_REDBACK },
+  {  1,  4,   500, FLAT, MONS_JUMPING_SPIDER },
+  {  1,  4,   900, FALL, MONS_JUMPING_SPIDER },
+  {  1,  4,   600, FLAT, MONS_TARANTELLA },
+  {  1,  4,   500, FALL, MONS_TARANTELLA },
+  {  1,  4,   400, FLAT, MONS_CULICIVORA },
+  {  1,  4,   500, FALL, MONS_CULICIVORA },
+  {  1,  4,   700, FLAT, MONS_DEMONIC_CRAWLER },
+  {  1,  4,   300, FALL, MONS_DEMONIC_CRAWLER },
+  // Harder enemies:
+  {  1,  4,   600, FLAT, MONS_WOLF_SPIDER },
+  {  1,  4,   400, RISE, MONS_WOLF_SPIDER },
+  {  1,  4,   400, FLAT, MONS_TORPOR_SNAIL },
+  {  1,  4,   200, RISE, MONS_TORPOR_SNAIL },
+  {  1,  4,   300, FLAT, MONS_ENTROPY_WEAVER },
+  {  1,  4,   300, RISE, MONS_ENTROPY_WEAVER },
+  {  1,  4,   400, FLAT, MONS_MELIAI },
+  {  1,  4,   400, RISE, MONS_MELIAI },
+  {  1,  4,   500, FLAT, MONS_PHARAOH_ANT },
+  {  1,  4,   400, RISE, MONS_PHARAOH_ANT },
+  {  1,  4,   200, FLAT, MONS_MOTH_OF_WRATH },
+  {  1,  4,   250, RISE, MONS_MOTH_OF_WRATH },
+  {  1,  4,   500, FLAT, MONS_STEELBARB_WORM },
+  {  1,  4,   300, RISE, MONS_STEELBARB_WORM },
+  {  1,  4,   200, FLAT, MONS_ORB_SPIDER },
+  {  1,  4,   350, RISE, MONS_ORB_SPIDER },
+  // Top-tier threats:
+  {  1,  4,   100, FLAT, MONS_GHOST_MOTH },
+  {  1,  4,   175, RISE, MONS_GHOST_MOTH },
+  {  1,  4,   100, FLAT, MONS_EMPEROR_SCORPION },
+  {  1,  4,   175, RISE, MONS_EMPEROR_SCORPION },
+  {  1,  4,   100, FLAT, MONS_BROODMOTHER },
+  {  1,  4,   150, RISE, MONS_BROODMOTHER },
+  {  1,  4,   75,  FLAT, MONS_JOROGUMO },
+  {  1,  4,   150, RISE, MONS_JOROGUMO },
+  {  1,  4,   50,  FLAT, MONS_RADROACH },
+  {  1,  4,   150, RISE, MONS_RADROACH },
+  {  1,  4,   100, FLAT, MONS_SUN_MOTH },
+  {  1,  4,   150, RISE, MONS_SUN_MOTH },
+  {  1,  4,   75,  FLAT, MONS_SPARK_WASP },
+  {  1,  4,   150, RISE, MONS_SPARK_WASP },
+  // OODs
```

Comment:
Spider doesn't actually have OOD monsters, does it? See 7b68f31d6f572688753d6cbcb57fac3b77f5d3b7 for an example of how to enable OODs, if you want them.

Also feels a bit sad that almost all of these are just high-end 'normal' spawns. Suppose there's not really any super late game stuff to steal, though. Should've added weird hell bugs...

Reply:
Sure, agree that shouldn't block merge. I was just rambling.